CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## Version 0.2.0 - released December 28, 2018
+
+- Geometry: Labeled polygon angles
+- Internal: Image storage refactor
+- Internal: Rollbar support
+- Internal: Cypress integration tests
+
+### Asset Sizes
+
+| File | Size | % Change from Previous Release |
+|---|---|---|
+| index.css | 318,419 bytes | 391% |
```

Comment:
It was from the incorporation of `blueprint.js`, which is used for the menu in the geometry tile. It's overkill for that purpose but since it was already being used for the Workspaces Table which is being ported to CLUE, I figured it made sense to use it rather than a standalone menu library or something like `material-ui`.